### PR TITLE
Add default introductory code example

### PR DESCRIPTION
**Pull request by kytrinyx**
Instead of having logic for a fallback in the backend, we're choosing a
default for the snippet file.

For tracks that have core exercises, we pick the first core exercise.
For tracks without these, we pick the first exercise listed in the config.

Note that we're aiming for 10 lines and a max-width of 40 columns.
This solution has a max-width of 43, so we may want to adjust it.

See https://github.com/exercism/meta/issues/89

### CODE REVIEW

File: docs/SNIPPET.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module HelloWorld where
+
+import Prelude
+import Data.Maybe (Maybe(Just, Nothing))
```
Comment (paf31) [inline docs/SNIPPET.txt]: I would replace this with `import Data.Maybe (Maybe(..))`

Otherwise, looks good to me!

Comment (paf31) [inline docs/SNIPPET.txt]: What's there should work fine though.

Comment (kytrinyx) [inline docs/SNIPPET.txt]: I'll make the switch, thanks!

Comment (kytrinyx) [inline docs/SNIPPET.txt]: Ok, done. Would you eyeball this to make sure it looks right? (Having never actually written any Purescript I fear the worst).

Comment (paf31) [inline docs/SNIPPET.txt]: Yes, it looks fine to me, thanks!

**Pull request merged**